### ISSUE TITLE
Ansible Galaxy won't install 'sardpost.kibana4' role from Galaxy

### ISSUE THREAD

**Issue by sardpost** [ansible/ansible#15351]
##### ISSUE TYPE

- Bug Report
##### ANSIBLE VERSION

```
1.9.4
```
##### CONFIGURATION

Defaults.
##### OS / ENVIRONMENT

Fedora 23 Linux 4.4.6-300.fc23.x86_64
##### SUMMARY

Ansible Galaxy does not install my sardpost.kibana4 role, but instead returns the error:

```
[root@dell-laptop sardpost]# ansible-galaxy install sardpost.kibana4
- downloading role 'kibana4', owned by sardpost
- downloading role from https://github.com/sardpost/sardpost.kibana4/archive/v0.1.0.tar.gz
- error: this role does not appear to have a valid meta/main.yml file.
- sardpost.kibana4 was NOT installed successfully.
- you can use --ignore-errors to skip failed roles.
```

The meta/main.yml seems fine. I double checked it many times. Deleted it, replaced it with a new one. The role gets imported fine in Galaxy with no errors but every time I try to install it, it gives me always the same error on meta/main.yml saying it's not valid.
##### STEPS TO REPRODUCE

Attempt to install sardpost.kibana4 role from Ansible Galaxy

```
ansible-galaxy install sardpost.kibana4
```

##### EXPECTED RESULTS

sardpost.kibana4 role is downloaded and installed correctly.
##### ACTUAL RESULTS

```
- downloading role 'kibana4', owned by sardpost
- downloading role from https://github.com/sardpost/sardpost.kibana4/archive/v0.1.0.tar.gz
- error: this role does not appear to have a valid meta/main.yml file.
- sardpost.kibana4 was NOT installed successfully.
- you can use --ignore-errors to skip failed roles.

```

**Comment by chouseknecht** [comment]
The issue is bad yaml found in your release v0.1.0 archive. If you open the archive and look at the yaml, you will notice 2 things:
1. It differs from the version found in the master branch. Galaxy (the web site) looks at the version found on the master branch, not the archive.
2. The yaml found in the release archive is missing a space on the very last line between 'dependencies:' and '[]'.  The line should actually read `dependencies: []`

Once I fixed the issue with the dependencies line, I was able to import the meta/main.yml file via python yaml.safe_load().
